test/run/t498.py.real.force

```diff
@@ -50,8 +50,8 @@ floating point and long integers
 0.0883883476483 <type 'float'>
 
 ERROR CHECKING:
-TypeError: unsupported operand type(s) for pow(): 'list' and 'str' on line 53
```

Comment:
do we still need this force file or do we now output the same as python?

Reply:
how can I check? is it only travis that does this or if I delete them on my machine will I be able to tell? I think yes to a most of them...

Reply:
i removed all the forced files associated with the tests I changed... travis likes it...